freshmaker/handlers/botas/botas_shipped_advisory.py

```diff
@@ -254,6 +254,10 @@ def handle(self, event):
                 pullspec_elems[1] = new_digest
                 new_pullspec = '@'.join(pullspec_elems)
                 pullspec['new'] = new_pullspec
+                # Always set pinned to True when it was replaced by Freshmaker
+                # since it indicates that the pullspec was modified from the
+                # original pullspec
+                pullspec['pinned'] = True
```

Comment:
We only replace the pullspec digest when `pinned` is True (https://github.com/redhat-exd-rebuilds/freshmaker/blob/4c0f1cab0e01d691665ce1841aa5738e74b47cfc/freshmaker/handlers/botas/botas_shipped_advisory.py#L237), so this is unnecessary.